src/packages/@ncigdc/modern_components/SampleType/SampleType.relay.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { graphql } from 'react-relay';
+import { compose, withPropsOnChange } from 'recompose';
+import Query from '@ncigdc/modern_components/Query';
```

Comment:
looks like the loading spinner that comes with the default export is too big for the table rows, export BaseQuery instead `import { BaseQuery } from '@ncigdc/modern_components/Query';`, it will pass a `loading` prop to the SampleType.js component, then render this GreyBox https://github.com/NCI-GDC/portal-ui/blob/next/src/packages/%40ncigdc/uikit/GreyBox.js when loading